playbooks/roles/openvpn/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ openvpn_clients:
 
 openvpn_port: "636"
 openvpn_port_udp: "8757"
+openvpn_port_sslh: "443"
 openvpn_key_size: "2048"
-openvpn_cipher: "AES-256-CBC"
+openvpn_cipher: "AES-256-GCM"
+openvpn_ncp_cipher: "AES-256-GCM:AES-256-CBC"
```

Comment:
Ah, the way you've done this would work for 2.3 clients as well. I didn't notice the `AES-256-CBC` here at first.